ansible/roles/ansible-tvault-contego-extension/tasks/nova_compute_filter_file.yml

```diff
@@ -1,6 +1,13 @@
+
 - name: Get nova venv path
+  shell: echo /openstack/venvs/nova-{{ venv_tag }}
+  register: venv_path
+  when: nova_virtual_env is undefined
+
+- name: Get nova venv path when nova_virtual_env is passed
   shell: echo {{ nova_virtual_env }}
   register: venv_path
+  when: nova_virtual_env is defined
 
```

Comment:
after setting the `venv_path`, check if that path exists.
if it does not, then fail the deployment

Reply:
resolved